compiler_gym/service/proto/event.proto

```diff
@@ -0,0 +1,51 @@
+// The CompilerGym service interface description.
+//
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+syntax = "proto3";
+
+package compiler_gym;
+
+option cc_enable_arenas = true;
+option go_package = "compiler_gympb";
+option java_multiple_files = true;
+option java_outer_classname = "EventProto";
+option java_package = "com.compiler_gym";
+
+import "basic.proto";
+import "google/protobuf/any.proto";
+
+message EventTuple {
+  repeated Event tuple = 1;
+}
+
+message EventDict {
+  map<string, Event> dict = 3;
+}
+
+// Common structure shared between Action and Observation.
+// TODO(boian): Maybe rename to Effect
```

Comment:
The `Event` message acts as a container for an unknown tensor or scalar. How about calling it `Tensor`?

Reply:
It is more than that. Because it can be a dictionary of events or it could be a `google.protobuf.Any`.
If only there was a good word to say that it could be an action or an observation.